tests/litmus-cleanup_test.go

```diff
@@ -0,0 +1,142 @@
+	scheme "k8s.io/client-go/kubernetes/scheme"
+	_ "k8s.io/client-go/plugin/pkg/client/auth/gcp"
+	"k8s.io/client-go/tools/clientcmd"
+
+	"github.com/litmuschaos/chaos-operator/pkg/apis/litmuschaos/v1alpha1"
+	chaosClient "github.com/litmuschaos/chaos-operator/pkg/client/clientset/versioned/typed/litmuschaos/v1alpha1"
+	restclient "k8s.io/client-go/rest"
+)
+
+var (
+	kubeconfig string
+	config     *restclient.Config
+	client     *kubernetes.Clientset
+	clientSet  *chaosClient.LitmuschaosV1alpha1Client
+)
+
+func TestChaos(t *testing.T) {
+
+	RegisterFailHandler(Fail)
+	RunSpecs(t, "BDD test")
+}
+
+var _ = BeforeSuite(func() {
+
+	var err error
+	kubeconfig = os.Getenv("HOME") + "/.kube/config"
+	config, err = clientcmd.BuildConfigFromFlags("", kubeconfig)
+
+	if err != nil {
+		Expect(err).To(BeNil(), "failed to get config")
+	}
+
+	client, err = kubernetes.NewForConfig(config)
+
+	if err != nil {
+		Expect(err).To(BeNil(), "failed to get client")
+	}
+
+	clientSet, err = chaosClient.NewForConfig(config)
+
+	if err != nil {
+		Expect(err).To(BeNil(), "failed to get clientSet")
+	}
+
+	err = v1alpha1.AddToScheme(scheme.Scheme)
+	if err != nil {
+		fmt.Println(err)
+	}
+
+})
+
+//BDD Tests to delete litmus
+var _ = Describe("BDD of litmus cleanup", func() {
+
+	// BDD TEST CASE 1
+	Context("Check for the litmus components", func() {
+
+		It("Should check for deletion of Litmus", func() {
+
+			//Deleting all chaosengines
+			var err error
+			By("Deleting all chaosengine")
+			err = exec.Command("kubectl", "delete", "chaosengine", "-n", "litmus", "--all").Run()
+			Expect(err).To(BeNil(), "failed to delete chaosengine")
+			if err != nil {
+				fmt.Println(err)
+			}
+
+			fmt.Println("All chaosengine deleted successfully")
+
+			//Deleting all chaosexperiment
+			By("Deleting all chaosexperiment")
+			err = exec.Command("kubectl", "delete", "chaosexperiment", "-n", "litmus", "--all").Run()
+			Expect(err).To(BeNil(), "failed to delete chaosexperiment")
+			if err != nil {
+				fmt.Println(err)
+			}
+
+			fmt.Println("All chaosexperiment deleted successfully")
+
+			//Deleting crds
+			By("Creating chaosengine crd")
+			err = exec.Command("kubectl", "delete", "-f", "https://raw.githubusercontent.com/litmuschaos/chaos-operator/master/deploy/chaos_crds.yaml").Run()
+			Expect(err).To(BeNil(), "failed to delete crds")
+			if err != nil {
+				fmt.Println(err)
+			}
+
+			fmt.Println("crds deleted successfully")
+
+			//Deleting rbacs
+			By("Creating chaosengine rbac")
+			err = exec.Command("kubectl", "delete", "-f", "https://raw.githubusercontent.com/litmuschaos/chaos-operator/master/deploy/rbac.yaml").Run()
+			Expect(err).To(BeNil(), "failed to create rbac")
+			if err != nil {
+				fmt.Println(err)
+			}
+
+			fmt.Println("rbac deleted sucessfully")
+
+			//Deleting chaos-operator deployment
+			By("Deleting chaos-operator deployment")
+			err = exec.Command("kubectl", "delete", "deploy", "-n", "litmus", "--all").Run()
```

Comment:
deleting all deployments or only chaos-operator?

Reply:
All @shubhamchaudhary  in all operator will also deleted